### ISSUE TITLE
dependency "packaging" used without definition in pipfile

### ISSUE THREAD

**Issue by frichtarik** [SergeyPirogov/webdriver_manager#454]
v3.8.4 fails because it requires packaging

but that module and it's version is not mentioned in pipfile

**Comment by yc5** [comment]
Got error message:
```
Traceback (most recent call last):
  ...
  from webdriver_manager.drivers.chrome import ChromeDriver
  File "/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/webdriver_manager/drivers/chrome.py", line 1, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
Error: Process completed with exit code 1.
```

**Comment by nodlesh** [comment]
This problem killed my test pipeline. Even doing `pip install packaging` caused chromium to stall in my linux container. Reverted back to 3.4.3.

**Comment by saidtahali** [comment]
I am also affected by this error, our testing jobs now are failing because of the packaging problem:
error message: 

`Traceback (most recent call last):
  File "/home/circleci/project/webdriver-installer.py", line 3, in <module>
    from webdriver_manager.****** import ChromeDriverManager
  File "/home/circleci/.pyenv/versions/3.9.7/lib/python3.9/site-packages/webdriver_manager/******.py", line 7, in <module>
    from webdriver_manager.drivers.****** import ChromeDriver
  File "/home/circleci/.pyenv/versions/3.9.7/lib/python3.9/site-packages/webdriver_manager/drivers/******.py", line 1, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'

Exited with code exit status 1`

**Comment by aleksandr-kotlyar** [comment]
fixed in #455
